test/unit/util/test-span.hpp

```diff
@@ -81,7 +81,14 @@ void testSpanIteratorTypes()
   using iterator = typename span_type::iterator;
   using const_iterator = typename span_type::const_iterator;
   IndexType len = 4;
-  ValueType* ptr = new ValueType[len]{0,1,2,3};
+  ValueType* ptr = new ValueType[len];
```

Comment:
This didn't use to be valid C++. Is it now valid? Or do we get lucky that compilers allow it.  Probably best to declare 'n' to be const or constexpr to be safe.

Reply:
Apparently, this new T {} syntax should be legal since C++11. I did not encounter this error with any other compiler.

I'm a bit puzzled as to why Travis did not catch this before.